demos/general/demo_pipeline.py

```diff
@@ -197,27 +198,29 @@ def main():
     cnm = cnmf.CNMF(n_processes, params=opts, dview=dview)
     cnm = cnm.fit(images)
 
-# %% ALTERNATE WAY TO RUN THE PIPELINE AT ONCE
+    # %% ALTERNATE WAY TO RUN THE PIPELINE AT ONCE (optional)
```

Comment:
At some point we'll need to clean up this phrasing more throughly (lowercase it too), but I'm not sure what "at once" is trying to say.

Reply:
It is explained a _little_ better in ipynb -- it runs motion correction, memory mapping, and cnmf in one method instead of breaking it up and running them separately. The py files are not a paragon of clarity. At some point we can loop around to actually improve them up my main goal here was to just make sure they work.

Reply:
Yeah, it's just weird phrasing. "AT ONCE"

Maybe "all in one go" is a better phrasing?